with-swift-graphql/README.md

```diff
@@ -0,0 +1,22 @@
+# GraphCMS x Swift with swift-graphql
+
+[Join our Slack](https://slack.graphcms.com)
+
+This demonstrates how to use GraphCMS with a Swift application for iOS (iPhone / iPad) or Mac, using popular library [swift-graphql](https://github.com/maticzav/swift-graphql)
```

Comment:
Were all issues resolved you ran into @craigtweedy? Maybe worth making a note below if there are any caveats.

Reply:
With regards to the schema generation? They weren't. 

I can make a note about it - will need to look into whether it's a problem with just this schema or wider. I'll have to raise the core issues with the maintainer as it's a good test bed anyway.